exp-models/addon/models/experiment.js

```diff
@@ -139,6 +139,21 @@ const Experiment = DS.Model.extend(JamModel, {
         maxDays = maxDays || Number.MAX_SAFE_INTEGER;
         return minDays <= age && age <= maxDays;
     },
+    isOldEnough(participant) {
```

Comment:
That would be great! Yes, this is just a pared-down isEligible, maybe it makes more sense to return in/out PLUS older/younger or -1/0/1 rather than a boolean here, or something else.